pkg/nuget/packages_props/parse.go

```diff
@@ -0,0 +1,92 @@
+package config
+
+import (
+	"encoding/xml"
+	"golang.org/x/xerrors"
+
+	dio "github.com/aquasecurity/go-dep-parser/pkg/io"
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+	"github.com/aquasecurity/go-dep-parser/pkg/utils"
+)
+
+	
+
+
+type propsPackageEntry struct{
+	Version         string   `xml:"Version,attr"`
+	UpdatePackageName     string   `xml:"Update,attr"`
+	IncludePackageName     string   `xml:"Include,attr"`
+}
+
+type propsPackageReferenceEntry struct {
+	XMLName         xml.Name `xml:"PackageReference"`
+	propsPackageEntry
+}
+
+type propsPackageVersionEntry struct {
+	XMLName         xml.Name `xml:"PackageVersion"`
+	propsPackageEntry
+}
+
+type propsItemGroup struct {
+	XMLName  xml.Name              `xml:"ItemGroup"`
+	ReferencePackages []propsPackageReferenceEntry `xml:"PackageReference"`
+	VersionPackages []propsPackageVersionEntry `xml:"PackageVersion"`
+}
+
+type propsProject struct {
+	XMLName xml.Name `xml:"Project"`
+	ItemGroups []propsItemGroup `xml:"ItemGroup"`
```

Comment:
ItemGroups looks as single tag. 
If you have seen `ItemGroups` array in docs or some examples - can you add comment about this and add that in tests?

Reply:
As could be seen in implementation there could be several:
https://github.com/dotnet/roslyn-tools/blob/380ffede53738d15ed1384e0f52dd8a3d937d38b/src/RoslynInsertionTool/RoslynInsertionTool/CoreXT.cs#L494C41-L494C41
Ill add more tests

Reply:
okay, thanks. Add this link to comment, please.